pydis_site/apps/api/serializers.py

```diff
@@ -250,6 +252,26 @@ class Meta:
         )
 
 
+class AocCompletionistBlockSerializer(ModelSerializer):
+    """A class providing (de-)serialization of `AocCompletionistBlock` instances."""
+
+    class Meta:
+        """Metadata defined for the Django REST Framework."""
+
+        model = AocCompletionistBlock
+        fields = ("user", "is_blocked", "reason")
```

Comment:
I'm not sure, but I think this is redundant since it's listing all fields. I believe leaving the fields unspecified would be equivalent.

Reply:
Doesn't this makes it a bit easier to understand?
In my personal opinion it makes it easier to understand what those serializers are for.

Reply:
I don't know. Maybe. But it's also yet another thing to maintain when a field gets changed. On the other hand, maybe we don't want new fields to implicitly be included in the serializer. Just a comment so I don't feel strongly either way.